CHANGES/3504.bugfix

```diff
@@ -0,0 +1 @@
+Fix type stubs for ``aiohttp.web.run_app(access_log=True)`` and fix edge case of ``access_log=True?? and the event loop being in debug mode.
```

Comment:
`??` need to replace with closing back quotes.

Reply:
oh yeah sorry, notebook keyboard :)